src/common/components/form/fields/MultiDropdownField.tsx

```diff
@@ -0,0 +1,88 @@
+import classNames from 'classnames';
+import { FieldProps } from 'formik';
+import { Select, SelectProps } from 'hds-react';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { invalidFieldClass } from '../constants';
+import { getErrorText } from '../utils';
+
+export type Option = {
+  label: string;
+  value: string;
+};
+
+type Props = SelectProps<Option> &
+  FieldProps & {
+    defaultValue: Option[];
+    setFieldValue?: (
+      field: string,
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      value: any,
+      shouldValidate?: boolean | undefined
+    ) => void;
+  };
+
+const DropdownField: React.FC<Props> = ({
+  className,
+  field: { name, onBlur, onChange, value, ...field },
+  form: { errors, touched },
+  helper,
+  multiselect,
+  options,
+  placeholder,
+  setFieldValue,
+  ...rest
+}) => {
+  const { t } = useTranslation();
+  const errorText = getErrorText(errors, touched, name, t);
+
+  const handleChange = (val: Option | Option[]) => {
+    const value = Array.isArray(val)
+      ? val.map((item) => item.value)
+      : val.value;
+    if (setFieldValue) {
+      setFieldValue(name, value);
+    } else {
+      onChange({
+        target: {
+          id: name,
+          value,
+        },
+      });
+    }
+
+    setTimeout(() => {
+      // Automatically call onBlur event to make formik set touched value to true
+      onBlur({
+        target: {
+          id: name,
+        },
+      });
+    });
+  };
+
+  return (
+    <Select
+      {...rest}
+      {...field}
+      helper={errorText || helper}
+      invalid={Boolean(errorText)}
+      optionLabelField={'label'}
```

Comment:
This is really minor, but I think that label is default value so this is redundant

Reply:
deleted prop